w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
+from datetime import datetime, timedelta
+
+MONDAYS = 1
+
+
+def pokemon():
+    pokemons = ['Bulbasaur', 'Squirtle', 'Charmander']
```

Comment:
Is this list going to change? if  not, it should be a tuple

Reply:
POKEMONS = ('Bulbasaur', 'Squirtle', 'Charmander')